AngularApp/projects/app-service-diagnostics/src/app/shared/components/tools/network-checks/network-check-flows/functionsFlow.js

```diff
@@ -516,9 +680,12 @@ async function validateConnectionStringAsync(propertyName, connectionString, typ
                     detailsMarkdown = title + `\r\n\r\n` + `This can be due to firewall rules on the resource.  Please check if you have configured firewall rules or a private endpoint and that they correctly allow access from the Function App.  Relevant documentation:`
                         + `\r\n\r\n`;
                     switch (type) {
-                        case ConnectionStringType.StorageAccount:

```

Comment:
what about legacy support based on isDaasNew here?

Reply:
I don't see the fix, could you plese explain?  We need to have some documentation for ConnectionStringType.StorageAccount as well when running against old daas version.